assets/merchant/getgems.json

```diff
@@ -47,6 +47,14 @@
             "tags": [],
             "submittedBy": "shuva10v",
             "submissionTimestamp": "2025-03-03T00:00:01Z"
+        },
+        {
+            "address": "EQBTGRpXgB3iQq74pHNcnKzFU_S4j4WGnubOiYYLU5JFOHwU",
+            "source": "tonviewer",
+            "comment": "nft_sale_getgems_v3",
```

Comment:
nft_sale_getgems_v3 is a special interface for sale contract. The address doesn't belong to this interface